docs/cli.md

```diff
@@ -22,6 +22,13 @@ You can also find information on flags with `promscale_<version> -help`
 | web-enable-admin-api | boolean | false | Allow operations via API that are for advanced users. Currently, these operations are limited to deletion of series. |
 | web-listen-address | string | `:9201` | Address to listen on for web endpoints. |
 | web-telemetry-path | string | `/metrics` | Web endpoint for exposing Promscale's Prometheus metrics. |
+| tls-cert-file | string | "" | TLS Certificate file for web server, leave blank to disable TLS.
+| tls-key-file | string | "" | TLS Key file for web server, leave blank to disable TLS.
+| basic-auth-username | string | "" | HTTP Basic Authentication Username used for web endpoint authentication. Disabled by default.
```

Comment:
```suggestion
| auth-username | string | "" | Authentication username used for web endpoint authentication. Disabled by default. |
```

Do we need to mention `basic auth` everywhere? I think it is obvious that username and password are for basic auth itself.